ic-os/dev-tools/bare_metal_deployment/deploy.py

```diff
@@ -472,15 +472,10 @@ def deploy_server(
             f"GetSetPowerStateREDFISH.py {cli_creds} -p {bmc_info.password} --set On",
         )
 
-        # If guestos ipv6 address is present, loop on checking connectivity.
-        # Otherwise, just wait.
         timeout_secs = 5
 
-        def wait_func() -> bool:
-            wait(timeout_secs)
-            return False
-
         def check_connectivity_func() -> bool:
+            log.info(f"Checking guestos ({bmc_info.guestos_ipv6_address}) connectivity...")
             assert bmc_info.guestos_ipv6_address is not None, "Logic error"
```

Comment:
Maybe we can collapse the optional nature of `guestos_ipv6_address`, while we're at it?